packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -75,15 +75,18 @@ export default class SaytDriverPlugin implements Plugin {
   }
 
   /**
-   * Dispatches an event with the response from the sayt data plugin.
+   * Dispatches an event with the response from the sayt data plugin and the origin searchbox id.
    * Callback for the Sayt data request event listener.
    *
    * @param event Event that contains the Sayt API request payload.
    */
   fetchSaytData(event: CustomEvent): void {
+    const searchbox = { searchbox: event.detail.searchbox };
```

Comment:
Please refer to my comment following this one: if you implement the feedback below then you won't need this variable. Since you're only using this in one spot I don't think you need to assign it to a variable.